CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required (VERSION 2.6)
 project(ale)
+set(ALEVERSION "0.5dev_b")
```

Comment:
The version question should be addressed now if possible, since this isn't too practical. See #147.